layers/modbus_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2017 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"fmt"
+	"github.com/google/gopacket"
+	"reflect"
+	"testing"
+)
+
+var (
+	opts = gopacket.Default
+)
+
+func init() {
+	opts.DecodeStreamsAsDatagrams = true
```

Comment:
instead of this, just initialize opts?

var opts = gopacket.DecodeOpts{DecodeStreamsAsDatagrams=true}